ste/xferRetryHelper_darwin.go

```diff
@@ -0,0 +1,25 @@
+//go:build darwin
+
+package ste
+
+import (
+	"errors"
+	"golang.org/x/sys/unix"
+	"net/http"
+	"strings"
+)
+
+func init() {
+	platformRetryPolicy = func(response *http.Response, err error) bool {
```

Comment:
since linux and darwin are so similar, can we group them together - we can add a comment saying if they do diverge in the future, we can split it into two files.